packages/scaffold-ui/src/partials/w3m-account-default-widget/index.ts

```diff
@@ -87,9 +88,12 @@ export class W3mAccountDefaultWidget extends LitElement {
         gap="l"
       >
         <wui-profile-button-v2
```

Comment:
If we still have `wui-profile-button` component, we could render that conditionally rather than using `wui-profile-button-v2` I think

Reply:
wui-profile-button doesn't allow to click and copy and correct icons, as far as I tested

Reply:
Okay yeah I guess they are different things